geth/node/node.go

```diff
@@ -185,6 +185,13 @@ func activateShhService(stack *node.Node, config *params.NodeConfig) error {
 			mailServer.Init(whisperService, whisperConfig.DataDir, whisperConfig.Password, whisperConfig.MinimumPoW)
 		}
 
+		if whisperConfig.LightClient {
```

Comment:
@dshulyak Please take a look. I think "please check out comment on when to set empty bloom filter" is resolved.

Reply:
i was sure i removed x-mark